### ISSUE TITLE
Unexpected error with Power BI

### ISSUE THREAD

**Issue by LosWochos76** [nimblelearn/datapackage-m#2]
When trying to execute the GDP example I get an unexpected error inside Power BI. I also tried different other URLs of core datasets but with no success.

The error-message states:
Microsoft.Mashup.Evaluator.Interface.ErrorException: The action is invalid due to the current state of the object.

I'm using Power BI version 2.67.5404.581 64-bit on my Windows 10 machine.

**Comment by michaelamadi** [comment]
Hi @LosWochos76,

Thanks for taking the time to report this. This is one of the [known issues](https://github.com/nimblelearn/datapackage-m/blob/5138fb55c48a62d29e1128f7ee1ff0d30d6764cb/README.md#known-issues) that we're currently working with the Power BI support team to resolve.

The issue only affects the `DataPackage.Tables` function so the current workaround is to use the `DataPackage.Table` function directly instead. An example of how to workaround the issue is provided [here](https://github.com/nimblelearn/datapackage-m/blob/5138fb55c48a62d29e1128f7ee1ff0d30d6764cb/README.md#known-issues).

Please grab the version of the functions from the latest release (v2.0.4 at the time of writing) and then use the 'Queries' pane to invoke the 'DataPackageTable' function using the 'Invoke Function' dialog or, if you're writing Power Query M expressions by hand, try something like this:

```text
let
    // Setup the shared function reference
    DataPackage.Table = DataPackageTable,

    // Invoke the function
    Source = DataPackage.Table("https://datahub.io/core/gdp/datapackage.json", null, "gdp")
in
    Source
```

Let me know how it goes 😃

**Comment by michaelamadi** [comment]
Hi @LosWochos76,

This issue has been fixed in release v2.0.5.
